### PR TITLE
Add StackedDenoisingAutoencoder

### PR DESCRIPTION
**Pull request by woozzu**
I deleted previous pull request and open a new request.
I tested it on the latest code.

### CODE REVIEW

File: pylearn2/models/autoencoder.py [hunk 1]
```diff
@@ -772,6 +772,46 @@ def _modify_updates(self, updates):
...
+        A list of autoencoder objects.
+    corruptor : object
+        Instance of a corruptor object to use for corrupting the
+        input.
+    """
+    def __init__(self, autoencoders, corruptor):
+        super(StackedDenoisingAutoencoder, self).__init__(autoencoders)
+        self.corruptor = corruptor
+
+    def reconstruct(self, inputs):
```
Comment (lamblin) [inline pylearn2/models/autoencoder.py]: You could just wrap `AbstractAutoencoder.reconstruct` instead of duplicating the docstring.

**Comment by lamblin** [comment]
Apart from that, it looks OK for me.
Can you let me know when you have updated the PR? Thanks!

**Comment by woozzu** [comment]
@lamblin I just fixed it.

**Comment by lamblin** [comment]
Great, thanks!

**Pull request merged**